### ISSUE TITLE
Add GitLens to VS Code Setup

### ISSUE THREAD

**Issue by rpatters1** [finale-lua/lua-scripts#248]
This is a proposal that we should add GitLens to the VS Code extensions list in our standard setup. It allows easy access to view and roll back changes from Git.

**Comment by Nick-Mazuk** [comment]
GitLens is a great extension. However, in my opinion, the recommended extensions should mainly be used for extensions specific to this repository (e.g., Lua support, formatting, etc.). And they should all be considered "essential" since VS Code will bug you every time you open the repo and don't have every recommended extension installed. So I don't think that place is the right place for nice-to-have but not essential extensions.

Personally, even though I've had GitLens installed in the past, I've found that I ended up using the command palette with the built-in Git functionality to do everything I needed.

For many of the more advanced GitLens allows you to do, I've found SourceGraph to be more useful:

https://sourcegraph.com/github.com/finale-lua/lua-scripts

Nowadays, I do almost everything related to code search or Git commit searching through SourceGraph.

However, since most developers coming to this repository likely have never coded before, I'd actually actively encourage a docs page in the "Getting started" category on creating a world-class developer environment in VS Code. We can mention many useful (though not essential) plugins on that page.

https://www.finalelua.com/docs/getting-started
